ten-simple-rules-dockerfiles.Rmd

```diff
@@ -119,10 +119,14 @@ Container images include the application, e.g., the programming language interpr
 A `Dockerfile` consists of a sequence of instructions to copy files and install software.
 Each instruction adds a layer to the image, which can be cached across image builds for minimizing build and download times.
 The images have a main executable exposed as an "entrypoint" that is started when they are run as stateful containers, which are the running instances of Docker images.
-Containers can be modified, stopped, restarted and purged.
+Containers can be modified, stopped, restarted and purged. A visual analogy with a compiled programming language is given in Figure&nbsp;1 for the workflow used to specify and build running Docker containers. 
```

Comment:
I love your image! Let's take this description a bit further and (even overly) describe it for the user. E.g., something like:

> A visual analogy for building a container is provided in Figure 1. Akin to compiling source code for a programming language, the container build also starts with a source file (Dockerfile) that provides instructions for building an image that runs to create a container instance, 

It probably seems overkill, but I think we should explain in detail so there isn't room for doubt.

Reply:
Thanks - I had a nice afternoon working on it :) 
I've commented on your follow-up below so we could resolve one or hopefully both of these now.